README.md

```diff
@@ -1 +1,23 @@
-# Setup-and-mobile-version-skeleton
\ No newline at end of file
+# Setup-and-mobile-version-skeleton
+
+> This is my portfolio Microverse project
+Additional description about the project and its features.
+
```

Comment:
Thanks for the suggestion, how exactly can i do this?

Reply:
You can add it with the help of GitHub pages, go to your application settings and on the left side choose to click on ` Pages ` and choose the branch you are going to create a live version, for this project keep the `/root` option for and click save, it will show you the link.